components/director/pkg/log/formatter_test.go

```diff
@@ -0,0 +1,80 @@
+package log
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Do we want to introduce a new testing framework for just a few tests? 

I think it is better to rewrite the tests with the standard go testing framework for consistency.